File: metadataviagens/Domain/Tripulantes/TipoTripulanteId.cs
using System;

namespace metadataviagens.Domain.Tripulantes
{
    public class TipoTripulanteId
    {
        public String id{get; private set; }

        public TipoTripulanteId(String id)
        {
            this.id=id;
        }
        
        public String toString(){
           return this.id;
        }
    }
}

File: metadatarede/src/domain/tipoTripulanteId.ts
import { Entity } from "../core/domain/Entity";
import { UniqueEntityID } from "../core/domain/UniqueEntityID";

export class TipoTripulanteId extends Entity<any> {

  get id (): UniqueEntityID {
    return this._id;
  }

  public static create (id?: UniqueEntityID): TipoTripulanteId {
    return new TipoTripulanteId(id);
  }
}